code/game/gamemodes/dynamic/dynamic_rulesets.dm

```diff
@@ -182,11 +182,6 @@
 /// Only called if ruleset is flagged as HIGHLANDER_RULESET
 /datum/dynamic_ruleset/proc/round_result()
 
-/// Checks if round is finished, return true to end the round.
-/// Only called if ruleset is flagged as HIGHLANDER_RULESET
-/datum/dynamic_ruleset/proc/check_finished()
-	return FALSE
-
```

Comment:
Not really sure about removing the possibility for rulesets to end the round completely. While this isn't used after this it does not mean it will stay unused, besides there is the ONLY_RULESET flag (which I have noticed won't actually work correctly because it can be picked after one ruleset has already been picked, this would be easy to fix though). Dynamic has a few functions which are unused that I added in case they might have a use later.

Reply:
dynamic shouldnt end the round outright based upon the results of a singular antag, it cucks all other antags and i personally dont agree with doing that. dynamic can have other ways for rounds to determine if it ends (nuke goes off, cult goes off etc.. but those can be hardcoded into their mechanic, since well... i dont think theres ever any reason not to end the round in those scenarios) elsewise admins can force end a round themselves if need be, or if admins arent on and theres SOME reason shuttle cant be called players can call a restart vote (though it would be cool if there was a shuttle call vote option, basically a softer restart that doesnt just bam, over, but is a more roleplay option thats for not super severe reasons (like stations in tact but the comms boards are gone and every method of obtaining one is gone, ie. lathe boards, spares and all, and circuit boards all cremated, perhaps a rare scenario but eh)